restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/ApplicationsDao.java

```diff
@@ -36,6 +36,9 @@
 
     List<Application> getApplications(Sid accountSid);
 
+    // this may optionally return related numbers as part of an Application
+    List<Application> getApplications(Sid accountSid, boolean includeNumbers);
```

Comment:
why dont we create a simple getApplicationsWithNumbers, who is goign to call getApplications(sid,false)??

Reply:
ok, @jaimecasero .